sesman/session.c

```diff
@@ -1199,6 +1204,33 @@ cleanup_sockets(int display)
         }
     }
 
+    /* the following files should be deleted by xorgxrdp
+     * but just in case the deletion failed */
+
+    g_snprintf(file, 255, XRDP_X11RDP_STR, display);
+    if (g_file_exist(file))
+    {
+        LOG(LOG_LEVEL_DEBUG, "cleanup_sockets: deleting %s", file);
+        if (g_file_delete(file) == 0)
+        {
+            LOG(LOG_LEVEL_DEBUG,
+                "cleanup_sockets: failed to delete %s", file);
```

Comment:
Couple of suggestions here:-
1. Use `LOG_LEVEL_WARNING` instead of `LOG_LEVEL_DEBUG` for the deletion failure.
2. Use something like `"cleanup_sockets: failed to delete %s: %s", file, g_get_strerror());` to log the error condition. This would apply to all the deletions in `cleanup_sockets()`

Reply:
Good point.